In [3]:
from google.colab import files
import pandas as pd
uploaded = files.upload()


Saving amazon_cells_labelled.txt to amazon_cells_labelled.txt


# Buddy@Move

In [5]:
dataset = pd.read_csv("buddymove_holidayi.csv")
dataset.describe()

Sports   Religious      Nature     Theatre    Shopping      Picnic
count  249.000000  249.000000  249.000000  249.000000  249.000000  249.000000
mean    11.987952  109.779116  124.518072  116.377510  112.638554  120.401606
std      6.616501   32.454115   45.639372   32.132696   41.562888   32.633339
min      2.000000   50.000000   52.000000   59.000000   50.000000   61.000000
25%      6.000000   84.000000   89.000000   93.000000   79.000000   92.000000
50%     12.000000  104.000000  119.000000  113.000000  104.000000  119.000000
75%     18.000000  132.000000  153.000000  138.000000  138.000000  143.000000
max     25.000000  203.000000  318.000000  213.000000  233.000000  218.000000

In [6]:
dataset.head()

User Id  Sports  Religious  Nature  Theatre  Shopping  Picnic
0  User 1       2         77      79       69        68      95
1  User 2       2         62      76       76        69      68
2  User 3       2         50      97       87        50      75
3  User 4       2         68      77       95        76      61
4  User 5       2         98      54       59        95      86

### Nome da Base
BuddyMove - buddymove_holidayiq

### Informação da Base
Este conjunto de dados foi preenchido a partir de revisões de destino publicadas por 249 revisores do site holidayiq.com até outubro de 2014. Foram consideradas as classificações em seis categorias entre os destinos no sul da Índia e a contagem de revisões em cada categoria para cada revisor (viajante).

### Fornecedor da Base:
Shini Renjith, shinirenjith@gmail.com

### Quantos atributos
6 atributos

### Tarefa
Classification, Clustering

### Tipos dos atributos
Atributo 1 : id do usuário - ordinal
Atributo 2 : Números de reviews sobre temas de esportes - ordinal
Atributo 3 : Números de reviews sobre temas de religião, templos, etc.  - ordinal
Atributo 4 : Números de reviews sobre temas de praias, rios, etc. - ordinal
Atributo 5 : Números de reviews sobre temas de teatros, museos, etc. - ordinal
Atributo 6 : Números de reviews sobre temas de compras, shopping, etc. - ordinal
Atributo 7 : Números de reviews sobre temas de parques, piquiniques, etc. - ordinal

### Informações publicadas
S. Renjith and C. Anjali, 'A personalized mobile travel recommender system using hybrid algorithm,' 2014 First International Conference on Computational Systems and Communications (ICCSC), Trivandrum, 2014, pp. 12-17. doi: 10.1109/COMPSC.2014.7032612

# News Aggregator

### Nome da Base
News Aggregator

### Informação da Base
As notícias são agrupadas em clusters que representam páginas que discutem a mesma notícia.
O conjunto de dados inclui também referências a páginas da Web que, no momento do acesso, apontavam (tem um link para) uma das páginas de notícias da coleção.

### Fornecedor da Base:
Provided by Artificial Intelligence Lab @ Faculty of Engineering, Roma Tre University - Italy 
Contato: Fabio Gasparetti, Faculty of Engineering, Roma Tre University - Italy (gaspare '@' dia.uniroma3.it)

### Quantos atributos
8 atributos

### Tarefa
Classification, Clustering

### Tipos dos atributos
ID	númerico
TITLE	- titulo da nóticia
URL - url da nóticia
PUBLISHER	- nome de quem publicou
CATEGORY - categoria da nóticia (b = negocios, t = ciencia e tecnologia, e = entretenimento, m = saúde) 
STORY	- cluster sobre nóticias da mesma história
HOSTNAME	- site que hospeda a nóticia
TIMESTAMP - tempo que a nóticia saiu

### Informações publicadas
Fabio Gasparetti. 2017. Modeling user interests from web browsing activities. Data Min. Knowl. Discov. 31, 2 (March 2017), 502-547.

# Sentiment Labelled Sentences

### Nome da Base
Sentiment Labelled Sentences

### Informação da Base
Contem sentenças e o sentimentos que elas produzem, negativo ou positivo.

### Fornecedor da Base:
Dimitrios Kotzias dkotzias '@' ics.uci.edu

### Quantos atributos
2 atributos

### Tarefa
Classification

### Tipos dos atributos
text - o texto analisado
class - 0 para sentimento negativo, 1 para positivo

### Informações publicadas
'From Group to Individual Labels using Deep Features', Kotzias et. al,. KDD 2015